src/factory.cpp

```diff
@@ -22,6 +22,9 @@ waybar::AModule* waybar::Factory::makeModule(const std::string& name) const {
     if (ref == "sway/window") {
       return new waybar::modules::sway::Window(id, bar_, config_[name]);
     }
+    if (ref == "sway/hide") {
+      return new waybar::modules::sway::Hide(id, bar_, config_[name]);
```

Comment:
Should that be a module?
Maybe we can implement it as a generic SwayBarConfigHandler class that is created by the `waybar::Client` instance when the SWAYSOCK is defined and controls all the bars.
The benefit is that on multi-monitor configurations bar events would be handled once.